docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -98,6 +98,224 @@ Interventions
+  \approx 2.22\: (0.95, 5.26).
+
+.. note::
+
+	We are modeling the reciprocal of the relative risk reported in the 
+	Cochrane review.
+
+	Additionally, this effect size crosses the null, and therefore, in some 
+	draws it will cause increasing coverage of the intervention to *increase* 
+	vitamin A deficiency prevalence. This is a limitation caused by the low 
+	quality evidence regarding the relative risk of vitamin A fortification on 
+	vitamin A prevalence. However, on average, increasing coverage of vitamin 
+	A fortification will decrease VAD.
+
+To model the uncertainty in this estimate, the above RR should be drawn from a
+`lognormal <https://en.wikipedia.org/wiki/Log-normal_distribution>`_
+distribution with median = 2.22, 2.5\ :superscript:`th`-percentile = 0.95, and
+97.5\ :superscript:`th`-percentile = 5.26. This distbibution can be created
+using `SciPy's lognorm function
+<https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.lognorm.html>`_
+as follows:
+
+.. code-block:: Python
+
+  from numpy import log
+  from scipy.stats import norm, lognorm
+
+  # median and 0.975-quantile of lognormal distribution for RR
+  median = 2.22
+  q_975 = 5.26
+
+  # 0.975-quantile of standard normal distribution (=1.96, approximately)
+  q_975_stdnorm = norm().ppf(0.975)
+
+  mu = log(median) # mean of normal distribution for log(RR)
+  sigma = (log(q_975) - mu) / q_975_stdnorm # std dev of normal distribution for log(RR)
+
+  # Frozen lognormal distribution for RR, representing uncertainty in our effect size
+  # (s is the shape parameter)
+  rr_distribution = lognorm(s=sigma, scale=median)
+
+.. note::
+
+	I copied this from Nathaniel's documentation for the folic acid RR, but I 
+	think that the same approach is appropriate. Perhaps we can eventually 
+	create a separate page that lists similar strategies that we can reference 
+	via links.
+
+Further, the time-to-response to vitamin A fortification in years should also 
+be sampled such that:
+
+.. code-block:: Python
+
+	from numpy import log
+	from scipy.stats import norm, lognorm
+
+	# median and 0.975-quantile of lognormal distribution for RR
+	median = 5/12
+	q_975 = 12/12
+
+	# 0.975-quantile of standard normal distribution (=1.96, approximately)
+	q_975_stdnorm = norm().ppf(0.975)
+
+	mu = log(median) # mean of normal distribution for log(RR)
+	sigma = (log(q_975) - mu) / q_975_stdnorm # std dev of normal distribution for log(RR)
+
+	# Frozen lognormal distribution for RR, representing uncertainty in our effect size
+	# (s is the shape parameter)
+	response_time_distribution = lognorm(s=sigma, scale=median)
+
+Additionally, as described in the research considerations above, the 
+intervention effect is dependent on age and time since intervention coverage.
+
+For simulants covered by *baseline coverage*, the effect of the 
+vitamin A fortitication is determined as follows:
+
+.. code-block:: Python
+
+  if age_i < 0.5:
+  	rr_i = 1
+  else:
+  	rr_i = rr_distribution.rvs()
```

Comment:
Aha, I see why I was confused by your description of rr - I think we are thinking about it differently.

The way this code is written, each simulant will get a different relative risk sampled from the lognormal distribution above. While this seems like a perfectly reasonable strategy, it is *not* what the lognormal distribution is intended to capture.

In particular, the lognormal distribution for the RR is intended to capture the *parameter uncertainty* of our estimate. That is, we imagine that there is one true value of the RR in the real world, but all these different studies measured different values that got combined in a meta-analysis, and we want to model the uncertainty in how far away we think the real value is from the average value reported by the meta-analysis. When we run the simulations, each draw should sample the lognormal distribution _once_ and use that value of the RR as the universal globally true value of the RR in the world simulated by that draw. (It is probably worth spending a little more time thinking about the theory of the statistics involved in parameter estimation to confirm that a lognormal distribution is in fact the appropriate thing to use here.)

On the other hand, it looks like you are using the lognormal distribution to model *heterogeneity between simulants*, i.e. each simulant will get a different random value for their relative risk. Usually our models have very little individual heterogeneity between simulants, because we don't have enough data to know what to do...

----
Ok, we just discussed this with James, but I'll go ahead and post this comment anyway.

Reply:
Yes, agreed to all of this. So I'll rework it to:

# draw level
rr = rr_distribution.rvs()

# individual level
 if age_i < 0.5:
  	rr_i = 1
  else:
  	rr_i = rr

Is this more clear/accurate?